plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.java

```diff
@@ -282,6 +282,8 @@
     public static String pref_page_database_resultsets_group_string;
     public static String pref_page_database_resultsets_checkbox_string_use_editor;
     public static String pref_page_database_resultsets_checkbox_string_use_editor_tip;
+    public static String pref_page_database_resultsets_label_replace_nulls_on_default;
+    public static String pref_page_database_resultsets_label_replace_nulls_on_default_tip;
```

Comment:
Looks like a typo. Did you mean to use `to` instead of `on`?

```suggestion
    public static String pref_page_database_resultsets_label_replace_nulls_to_default;
    public static String pref_page_database_resultsets_label_replace_nulls_to_default_tip;
```